source/deployment/sso-saml-technical.rst

```diff
@@ -0,0 +1,109 @@
+              </samlsig:X509Data>
+          </samlsig:KeyInfo>
+      </samlsig:Signature>
+  </samlp:AuthnRequest>
+
+SAML Responses
+------------------------------------------------------------
+
+There are different types of SAML responses sent by the IdP to the SP. The response contains the Assertion with the NameID and attributes of a user.
+
+Below is a table of the different types of responses. Each response type is fully supported except when the SAML assertion is signed while the SAML response itself is not:
+
+.. csv-table::
+    :header: "Response Type", "Signed SAML Response", "Signed SAML Assertion", "Encrypted SAML Assertion", "Supported by Mattermost"
+
+    "Type 1", "Yes", "Yes", "Yes", "Yes"
+    "Type 2", "Yes", "Yes", "No", "Yes"
+    "Type 3", "Yes", "No", "Yes", "Yes"
+    "Type 4", "Yes", "No", "No", "Yes"
+    "Type 5", "No", "Yes", "Yes", "Partially, validation of assertion signature not supported"
+    "Type 6", "No", "Yes", "No", "Partially, validation of assertion signature not supported"
+    "Type 7", "No", "No", "Yes", "Yes ??"
```

Comment:
```suggestion
    "Type 7", "No", "No", "Yes", "Yes?"
```

Reply:
@enahum Meant to confirm -- do we support an unsigned SAML request with an unsigned BUT encrypted SAML assertion? I believe it's a yes.

Reply:
Yes